src/GongSolutions.WPF.DragDrop/DefaultDropHandler.cs

```diff
@@ -220,10 +226,22 @@ public virtual void Drop(IDropInfo dropInfo)
                         {
                             obj2Insert = cloneable.Clone();
                         }
-                    }
+                    }                   
```

Comment:
Should avoid adding whitespace to the end of lines.
```suggestion
                    }
```